data/json/bionics.json

```diff
@@ -358,54 +565,108 @@
+            [ "HAND_R", 1 ],
+            [ "LEG_L", 4 ],
+            [ "LEG_R", 4 ],
+            [ "FOOT_L", 1 ],
+            [ "FOOT_R", 1 ]
+        ],
         "act_cost": 50,
         "description": "At an immense cost of power, you may increase your body speed and reactions dramatically, essentially freezing time.  You are still delicate, however, and violent or rapid movements may damage you due to friction."
     },{
         "type": "bionic",
         "id": "bio_teleport",
         "name": "Teleportation Unit",
+        "occupied_bodyparts": [
+            [ "TORSO", 16 ],
+            [ "ARM_L", 3 ],
+            [ "ARM_R", 3 ],
+            [ "LEG_L", 4 ],
+            [ "LEG_R", 4 ]
+        ],
         "act_cost": 250,
         "description": "This highly experimental unit folds space over short distances, instantly transporting your body up to 25 feet at the cost of much power.  Note that prolonged or frequent use may have dangerous side effects."
     },{
         "type": "bionic",
         "id": "bio_blood_anal",
         "name": "Blood Analysis",
+        "occupied_bodyparts": [
+            [ "TORSO", 8 ]
+        ],
         "act_cost": 25,
         "description": "Small sensors have been implanted in your heart, allowing you to analyze your blood.  This will detect many illnesses, drugs, and other conditions."
     },{
         "type": "bionic",
         "id": "bio_geiger",
         "name": "Integrated Dosimeter",
+        "occupied_bodyparts": [
+            [ "TORSO", 2 ],
+            [ "ARM_L", 1 ],
+            [ "ARM_R", 1 ],
+            [ "LEG_L", 1 ],
+            [ "LEG_R", 1 ]
+        ],
         "act_cost": 25,
         "description": "Small radiation sensors have been implanted throughout your body, allowing you to analyze your level of absorbed radiation.  They will also alert you whenever exposed to environmental radiation."
     },{
         "type": "bionic",
         "id": "bio_radscrubber",
         "name": "Radiation Scrubber System",
+        "occupied_bodyparts": [
+            [ "TORSO", 5 ],
+            [ "ARM_L", 1 ],
+            [ "ARM_R", 1 ],
+            [ "LEG_L", 2 ],
+            [ "LEG_R", 2 ]
+        ],
         "act_cost": 50,
         "description": "A system of advanced piezomechanical blood filters have been implanted throughout your body, allowing you to purge yourself of absorbed radiation at the cost of some bionic power."
     },{
         "type": "bionic",
         "id": "bio_adrenaline",
         "name": "Adrenaline Pump",
+        "occupied_bodyparts": [
+            [ "TORSO", 6 ]
+        ],
         "act_cost": 50,
         "description": "A stimulator system has been implanted alongside your adrenal glands, allowing you to trigger your body's adrenaline response at the cost of some bionic power."
     },{
         "type": "bionic",
         "id": "bio_blood_filter",
         "name": "Blood Filter",
+        "occupied_bodyparts": [
+            [ "TORSO", 6 ]
```

Comment:
Less than blood analysis? It has a much more powerful (and much less subtle) function.